.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+
+**Is this a BUG REPORT, PERFORMANCE REPORT or FEATURE REQUEST?**:
+
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+**Anything else we need to know?**:
+
+**Environment**:
+- Kubernetes version (use `kubectl version`):
+- Gimbal version:
```

Comment:
How does someone figure out which version of Gimbal they're running?  What about the version of Contour?

Reply:
Hmm great question. We don't have a Gimbal version anywhere other than the git repository.

Might be better to ask for Contour version and discoverer versions.